From ee4ae07de7a51b47fd5c0695c6749815cd6d1e50 Mon Sep 17 00:00:00 2001
From: Laurent Defert <laurent_defert@yahoo.fr>
Date: Sun, 7 Aug 2011 12:43:14 +0200
Subject: [PATCH] Fix large files support

Put back AC_SYS_LARCFILE that was removed during
70e05fa6d033eeebaf7e072233b45d334d6139d8 to fix large file support
---
 configure.ac |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/configure.ac b/configure.ac
index fe700b9..5d2ebfc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -122,6 +122,10 @@ AC_MSG_RESULT([$have_cpuid])
 
 AC_CHECK_SIZEOF([long])
 
+dnl Support large files / 64 bit seek offsets.
+dnl Use --disable-largefile if you don't want this.
+AC_SYS_LARGEFILE
+
 dnl Availability of various common functions (non-fatal if missing),
 dnl and various less common threadsafe functions
 AC_CHECK_FUNCS_ONCE([cfmakeraw geteuid getgid getgrnam_r getmntent_r \
-- 
1.7.3.4

